backend/src/models/users/user.type.ts

```diff
@@ -42,6 +42,10 @@ export class User extends Model {
   @HasOne(() => Login)
   public login!: Login;
 
+  @Field(() => Boolean)
+  @Column(DataType.BOOLEAN)
+  public isPaidAccount!: boolean;
+
```

Comment:
Payments will be its own section of code that is brought in once we have the API feature stable and we have some time to understand the best path forward. Payments while an important feature will come in due time, it will also maybe be a module that is enabled and outside of the core open source software package.

The other thing to note is it wont just be isPaidAccount but will be multiple columns in its own table. Example of this table 

```
id -> UUID
userId -> UUID
planName -> String
planPrice -> Float (Sometimes used for legacy, but nullable)
planProjectLimit -> Number
renewalDate -> Date
stripeId -> String 
```

This should give us enough data when the time comes, this is just a quick thought and could change in the future depending on needs.

Reply:
Anything I can do to get started on that?

Reply:
https://github.com/authsio/core/pull/17#issuecomment-1027400100

Lets talk, to sync on this.

Reply:
The key here is 

```
 it will also maybe be a module that is enabled and outside of the core open source software package.
```